todo/services/task_service.py

```
@@ -221,3 +221,45 @@ def create_task(cls, dto: CreateTaskDTO) -> CreateTaskResponse:
                     ],
                 )
             )
+
+    @staticmethod
+    def add_label_to_task(task_id: PyObjectId, label_id: PyObjectId) -> None:
```

Ruff (0.11.9)

Undefined name `PyObjectId`

(F821)